### PR TITLE
Disabled Playbooks are no Longer Visible in the UI

### PR DESCRIPTION
**Pull request by AshirRashid**
# Description

Disabled Playbooks are no Longer Visible in the UI (#1304)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue).

# Checklist

- [x] I have read and understood the rules about [how to Contribute](https://intelowl.readthedocs.io/en/latest/Contribute.html) to this project
- [x] The pull request is for the branch `develop`
- [x] The tests gave 0 errors.
- [x] Linters (`Black`, `Flake`, `Isort`) gave 0 errors. If you have correctly installed [pre-commit](https://intelowl.readthedocs.io/en/latest/Contribute.html#how-to-start-setup-project-and-development-instance), it does these checks and adjustments on your behalf.
- [x] If the GUI has been modified, I have a provided a screenshot of the result in the PR.
The Disabled Playbook is no longer selectable in the scan form:
CleanShot 2023-01-27 at 01 31 09@2x

### Important Rules
- If you miss to compile the Checklist properly, your PR won't be reviewed by the maintainers.
- If your changes decrease the overall tests coverage (you will know after the Codecov CI job is done), you should add the required tests to fix the problem
- Everytime you make changes to the PR and you think the work is done, you should explicitly ask for a review. After being reviewed and received a "change request", you should explicitly ask for a review again once you have made the requested changes.

### CODE REVIEW

**Comment by AshirRashid** [comment]
Running the command `black . --check --diff --exclude "migrations|venv"` leaves all files unchanged, but it returns an error int he test. The error is in a file that I haven't edited.

File: frontend/src/components/scan/ScanForm.jsx [hunk 1]
```diff
@@ -250,6 +250,7 @@ export default function ScanForm() {
     () =>
       playbooksGrouped[classification]
         .map((v) => ({
+          isDisabled: !v.verification.configured || v.disabled,
```
Comment (mlodic) [inline frontend/src/components/scan/ScanForm.jsx:253]: this is copy/pasted from analyzers/connectors. Playbooks do not have `!v.verification.configured` value. It is enough to set the second clause

**Comment by 0ssigeno** [comment]
The black error was my fault, you should be able to rebase `develop` and have the error fixed!

**Review by mlodic — CHANGES_REQUESTED** [review]

**Comment by mlodic** [comment]
ty!

**Pull request merged**